src/websocket.c

```diff
@@ -56,6 +56,8 @@ static const uint8_t WS_HEADER_FIN = 0x80;
 #define WS_PING_FRAME 0x9
 #define WS_PONG_FRAME 0x0a
 
+#define WS_SMALL_FRAME_SIZE 125
```

Comment:
If possible use static const unsigned int instead of a define. This leads to a symbol which is helpful while debugging.